### PR TITLE
bug fix - move sampleRate field out from tag to root

### PR DESCRIPTION
**Pull request by xiao-lix**

### CODE REVIEW

File: AISKU/Tests/TelemetryValidation/CommonValidator.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { ITypeValidator } from './ITypeValidator';
-import { IEnvelope } from '@microsoft/applicationinsights-common';
+import { IEnvelope, SampleRate } from '@microsoft/applicationinsights-common';
```
Comment (markwolff) [inline AISKU/Tests/TelemetryValidation/CommonValidator.ts]: Is this being used in this file?

File: AISKU/Tests/applicationinsights.e2e.tests.ts [hunk 2]
```diff
@@ -82,6 +85,8 @@ export class ApplicationInsightsTests extends TestClass {
             this.successSpy = this.sandbox.spy(sender, '_onSuccess');
             this.loggingSpy = this.sandbox.stub(this._ai['core'].logger, 'throwInternal');
             this.trackSpy = this.sandbox.spy(this._ai['dependencies'], 'trackDependencyDataInternal')
+            this.isSampledInSpy = this.sandbox.stub(sender, '_isSampledIn', () => true);
```
Comment (markwolff) [inline AISKU/Tests/applicationinsights.e2e.tests.ts]: I don't think this var is used anywhere, so you can just not assign this to a variable

File: channels/applicationinsights-channel-js/Tests/Sample.tests.ts [hunk 3]
```diff
@@ -31,6 +31,7 @@ export class SampleTests extends TestClass {
 
                 Assert.ok(this.sample.isSampledIn(this.item));
                 Assert.ok(scoreStub.notCalled);
+                Assert.ok(!this.item[SampleRate], "sampleRate field is hidden when the sampling percentage is 100");
```
Comment (markwolff) [inline channels/applicationinsights-channel-js/Tests/Sample.tests.ts]: Does this test  fail on the other test cases with a <100 sampling rate?

Comment (xiao-lix) [reply channels/applicationinsights-channel-js/Tests/Sample.tests.ts]: Thought about it, it actually doesn't make sense to test it here. SampleRate field will be added in the envelope in "processTelemetry" method of Sender, then serialized off if it equals to 100 in the same function. so removed from here.

**Pull request merged**